actionview/lib/action_view/renderer/template_renderer.rb

```diff
@@ -50,6 +50,7 @@ def render_template(template, layout_name = nil, locals = nil) #:nodoc:
       view, locals = @view, locals || {}
 
       render_with_layout(layout_name, locals) do |layout|
+        ActiveSupport::Notifications.instrument("start_rendering.action_view", identifier: template.identifier, layout: layout.try(:virtual_path))
```

Comment:
I don't like the idea of publishing a new instrumentation when we already know where a instrumentation starts. Our Subscribers are evented, it works with `start` and `finish` events. We only need to expose an API for this in the subscribers. I'll work on that.